docs/assets/templates/tools/barebones-.dockstore.yml

```diff
@@ -0,0 +1,10 @@
+version: 1.2
+tools:
+  - subclass: <CWL | WDL | NFL | GALAXY>
+    primaryDescriptorPath: <String>
+    testParameterFiles: <String Array>
+    authors:
+      - name: <String>
+        orcid: <String>
+        email: <String>
+    name: <String>
```

Comment:
without looking at where this is linked to, this isn't quite `barebones` since name, author are optional. 
Probably a bad file name, so probably ignore. Will quit commenting on this

Reply:
I purposely include all three because:
1. Name is required if orcid is not provided so I can't get rid of name if we include authors at all
2. I wanted to "show off" the orcid option
3. I wanted to include email (and authors overall) to encourage people to have contact information for their workflows/tools/services

That does mean that it strictly is not barebones, but it is still simple. How do you feel about it? Perhaps I should get rid of the orcid one though.

Reply:
I had thought about commenting on this before but passed, now seeing the discussion I'm drawn in :)

If the name is going to be "barebones", then it might end with line 4, the primaryDescriptorPath. And/or perhaps rename to "minimal"

Most people don't use authors at all, although maybe this is a good place to start, but it always optional, whereas name is required if there is more than one entry. IOW, I would put name before authors.

Reply:
I actually meant the `name` on line 10 is optional (if you only have one tool or workflow in the repo), but the orcid name is optional of course too. But I do feel strongly that if it is called barebones, it should have the absolute minimum (only the required fields). 

That said, I do really appreciate this exercise + examples. You could have 

*  barebones (the absolute minimum)
* minimal (what the current barebones is)
* recommended
* maximum (the really well annotated ones you have already)

Reply:
> I wanted to "show off" the orcid option

`orcid` should be a separate author technically since we get info like name and email from their ORCID record. I lean against including `orcid` with the other author fields to avoid confusion about where the author info is coming from. In this example, the .dockstore.yml `name` and `email` will be ignored when the user views the author information in the UI

Reply:
Ultimately I am somewhat against a truly absolute minimum example, because I think it is important to encourage people to at the very least include some sort of contact via authorship information. We've already run into some issues where it isn't clear who is responsible for a Dockstore entry.

Of course, maybe that's me trying to kill two birds with a stone that really should only be used to kill one bird. Thoughts on this? I heard user profiles are being explored so perhaps this will become moot.

Reply:
I suggest removing barebones and minimal and instead focusing on recommended and maximum to ensure we have high-quality metadata.

Reply:
I do think that one of barebones and minimal is nice as a software developer. I usually try to get things hooked up with something along those lines and having to fill out everything recommended right off the bat can be off-putting.